client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -19,7 +24,19 @@ const ConnectedSingleSiteWpcom = connectOptions( ( props ) => {
 	const upsellUrl = `/plans/${ siteSlug }`;
 	let upsellBanner = null;
 	if ( displayUpsellBanner ) {
-		upsellBanner = (
+		upsellBanner = isEnabled( 'themes/premium' ) ? (
+			<UpsellNudge
+				className="themes__showcase-banner"
+				event="calypso_themes_list_premium_themes"
+				feature={ FEATURE_PREMIUM_THEMES }
+				plan={ PLAN_BUSINESS }
+				title={ translate(
+					'Unlock all premium themes with our Premium, Business and eCommerce plans!'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Upload your own themes with our Premium and Business plans!' )` *ES Score: 9*

Reply:
I think we'll want to break this into multiple upsells rather than combining them /cc @ianstewart to confirm.

For Free or Personal plan sites:

> Unlock ALL premium themes with our Premium and Business plans!

For Premium plan sites:

> Upload your own themes with our Business and eCommerce plans!